### DISCUSSION TITLE
[BUG]supervisord

### DISCUSSION THREAD

**Discussion by iygwei** [1Panel-dev/1Panel#7079] [Q&A]
### 联系方式

iygwei@qq.com

### 1Panel 版本

1.10.20-lts

### 问题描述

https://hub.docker.com/layers/1panel/php/8.0.30-fpm/images/sha256-fbf530310bf0e95ef6d78d772bd6b67884f6623683317dffd7a3db0cade7c37f?context=explore

我看到这里面存在supervisord, 但是我无法在容器里面执行

### 重现步骤

进入容器

**Comment by wanghe-fit2cloud** [comment]
这是 1Panel 自行维护的 PHP 基础镜像，将在 v2 版本中使用。
目前 v1 版本的 PHP 运行环境仍然基于官方提供的镜像。

**Reply by iygwei** [reply]
v2 怎么下载测试版也成

**Reply by wanghe-fit2cloud** [reply]
还不支持直接在 v1 版本中使用，因为底层安装扩展部分都重构了。
目前你可以在 1Panel 论坛中找一下相关帖子。
